### PR TITLE
Alpha diversity boxplot example

### PR DESCRIPTION
**Pull request by TuomasBorman**
Hi,

here are an example of boxplot that visualizes Shannon index, and an example of pairwise Wilcoxon test.

-Tuomas

### CODE REVIEW

File: 14a_alpha_diversity.Rmd [hunk 1]
```diff
@@ -109,6 +109,29 @@ se <- mia::estimateDiversity(se,
...
+plotting them as points. With more data points, test `geom_jitter` function
+instead of `geom_point`.
+
+```{r visualize-shannon}
+library(ggplot2)
+ggplot(as.data.frame(colData(se)), aes(x = SampleType, y = shannon)) +
+  # Outliers are removed, because otherwise each data point would be plotted twice; 
+  # as an outlier of boxplot and as a point of dotplot.
+  geom_boxplot(outlier.shape = NA) + 
+  geom_point() +
```
Comment (antagomir) [inline 14a_alpha_diversity.Rmd:123]: wouldn't `geom_jitter` work better? The jitter width could be set to 0.5 or something suitable.

File: 14a_alpha_diversity.Rmd [hunk 2]
```diff
@@ -109,6 +109,29 @@ se <- mia::estimateDiversity(se,
...
+```
+To test, if these differences that can be seen in plot are statistically significant
+we do pairwise Wilcoxon test. It compares every group with each others to see if 
+their values differ. 
+
+```{r pairwise-wilcoxon}
+df <- as.data.frame(colData(se))
+pairwise.wilcox.test(df$shannon, df$SampleType,  p.adjust.method = "fdr")
+```
+
```
Comment (antagomir) [inline 14a_alpha_diversity.Rmd:134]: Would be ideal to show the differnces in the figure itself using `ggsignif` or other suitable packages?

File: 14a_alpha_diversity.Rmd (hunk 1 above)
Comment (TuomasBorman) [reply 14a_alpha_diversity.Rmd]: Changed to jitter

File: 14a_alpha_diversity.Rmd (hunk 2 above)
Comment (TuomasBorman) [reply 14a_alpha_diversity.Rmd]: Removed wilcoxon and added geom_signif

**Comment by antagomir** [comment]
Thanks

**Pull request merged**